src/test/java/io/stargate/sgv2/jsonapi/api/v1/AbstractKeyspaceIntegrationTestBase.java

```diff
@@ -274,4 +281,29 @@ protected float[] decodeBase64BinaryVectorToFloatArray(String base64Vector) {
       throw new RuntimeException("Not a valid binary vector", e);
     }
   }
+
+  protected boolean executeCqlStatement(SimpleStatement statement) {
```

Comment:
Idea: maybe create/change to take `SimpleStatement ...` to allow multiple statements to be executed within same session? This would avoid overhead of multiple separate sessions if multiple statements are always needed to initialize something.

Reply:
thanks, fixed